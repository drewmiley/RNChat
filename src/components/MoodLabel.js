import React, { Component } from 'react';
import {
	Text,
	View
} from 'react-native';

export default class MoodLabel extends Component {
    constructor(props) {
        super(props);
        this.shouldComponentUpdate = false;
    };
    render() {
        return <View>
            <Text>DREW is {this.props.mood}</Text>
        </View>
    };
};