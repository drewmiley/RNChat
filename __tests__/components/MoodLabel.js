import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import MoodLabel from '../..//src/components/MoodLabel';

describe('MoodLabel', () => {

    describe('Rendering', () => {

        it('should match the snapshot', () => {
            const component = renderer.create(
                <MoodLabel />
            );            
            expect(component).toMatchSnapshot();
        });

    });

});