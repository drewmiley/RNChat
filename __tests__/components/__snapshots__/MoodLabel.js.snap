exports[`MoodLabel Rendering should match the snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    DREW is 
  </Text>
</View>
`;
